### PR TITLE
Add Events table createDateTime field & Seed data

### LINKED ISSUE #46: Need a Create Date/Time field in the Events table

### PR DESCRIPTION
**Pull request by PaulMEdwards**
Add Events table createDateTime field & Seed data;
Replicate PR 43 changes here.

Tests probably need to be updated to account for the new column.

# Description

Fixes #46 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data/migrations/20200410195204_neighborhood-chef.js [hunk 1]
```diff
@@ -37,14 +38,16 @@ exports.up = function(knex) {
       tbl.integer('event_id').notNullable().unsigned().references('Events.id').onUpdate('CASCADE').onDelete('CASCADE');
       tbl.integer('user_id').notNullable().unsigned().references('Users.id').onUpdate('CASCADE').onDelete('CASCADE');
       tbl.integer('inviter_id').notNullable().unsigned().references('Users.id').onUpdate('CASCADE');
-      tbl.enum('status', [ 'Not Approved', 'Approved', 'Not Going', 'Maybe Going', 'Going' ]).notNullable().defaultTo('false');
+      tbl.enum('status', [ 'Not Approved', 'Approved', 'Not Going', 'Maybe Going', 'Going' ]).notNullable().defaultTo('Not Approved');
```
Comment (PaulMEdwards) [inline data/migrations/20200410195204_neighborhood-chef.js:41]: #43 changes

File: data/migrations/20200410195204_neighborhood-chef.js [hunk 2]
```diff
@@ -37,14 +38,16 @@ exports.up = function(knex) {
...
       tbl.primary(['event_id', 'user_id']);
     })
     ;
 };
 
 exports.down = function(knex) {
   return knex.schema
+    .dropTableIfExists('Events_Attending')
+    .dropTableIfExists('Events_Invited')
```
Comment (PaulMEdwards) [inline data/migrations/20200410195204_neighborhood-chef.js:50]: #43 changes

**Review by kyle-richardson — APPROVED** [review]
thanks. looks good

**Review by bvneilson — APPROVED** [review]
Thank you Paul!!

**Pull request merged**